api/server/server.go

```diff
@@ -20,12 +20,20 @@ import (
 
 type ToDDApi struct {
 	cfg config.Config
+	tdb db.DatabasePackage
 }
 
-func (tapi ToDDApi) Start(cfg config.Config) {
+func (tapi ToDDApi) Start(cfg config.Config) error {
 
 	tapi.cfg = cfg
 
+	tdb, err := db.NewToddDB(tapi.cfg)
+	if err != nil {
+		return err
```

Comment:
Okay so after I went through and looked at where I previously commented, I confirmed this is just insufficient logging on my part. So I created https://github.com/Mierdin/todd/issues/6 to track this, and don't worry about it for now. It was a problem before you added these errors.